rpm/poppler.spec

```diff
@@ -142,6 +153,12 @@ rm -f %{buildroot}%{_libdir}/*.la
 %{_libdir}/libpoppler-cpp.so
 %{_includedir}/poppler
 
+%files private-devel
```

Comment:
The above devel package includes already %{_includedir}/poppler so it has all the content that's listed below. Should %exclude there what's not supposed to go to the package.

I'm noticing there's already a similar problem with the qt5-devel subpackage.

Reply:
Do we have to have a separate package for these? Neither Fedora nor openSUSE have separate packages for these. 

These seem to be entirely opt-in.

Reply:
This starts to be getting a bit complicated, and I wouldn't be that excited on going through the file list whenever updating poppler. Maybe we should just include these in the main -devel package. There doesn't seem to be other side-effects than installation of the files, and app developer should have some idea from other places on what APIs should be used.

Out of curiousity, what's the thing or feature that needs the unstable APIs? The stable ones should be having quite a lot of features supported already.

Reply:
In all fairness the list of official .h files is very short.
I really liked the package split (hoping the official part can be OK:ed in Harbour someday, and the split being an extra help to not accidentally use private apis).
But after all it is your packaging.